packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -29,7 +29,9 @@ export const Pages = {
   },
   AddDataSource: {
     url: '/datasources/new',
+    /** @deprecated */
     dataSourcePlugins: (pluginName: string) => `Data source plugin item ${pluginName}`,
+    dataSourcePlugins2: (pluginName: string) => `Add data source ${pluginName}`,
```

Comment:
I think we are using this other conversion based on this [PR](https://github.com/grafana/grafana/pull/41787):

```suggestion
    dataSourcePluginsV2: (pluginName: string) => `Add data source ${pluginName}`,
```

Reply:
Ah, nice catch. 👍